build/pr-netfx-validation.yaml

```diff
@@ -59,12 +59,12 @@ jobs:
       outputDirectory: 'TestResults'
       artifactName: 'netfx_tests_windows-$(Build.BuildId)'
 
-  - template: azure-pipeline.template.yaml
-    parameters:
-      name: 'netfx_451_tests_windows'
-      displayName: '.NET Framework 451 Unit Tests (Windows)'
-      vmImage: 'vs2017-win2016'
-      scriptFileName: build.cmd
-      scriptArgs: RunTestsNetFx451 incremental
-      outputDirectory: 'TestResults'
-      artifactName: 'netfx_451_tests_windows-$(Build.BuildId)'
+  # - template: azure-pipeline.template.yaml
```

Comment:
If this is commented, then:
- Do we plan to bring it back , if so then let's add link to work item to ensure we don't forget
- If not, let's kill it

Although I think this is only temporary.

Reply:
The following bugs were logged to fix these issues:
- [Bug 26682588](https://msazure.visualstudio.com/One/_workitems/edit/26682588): Gateway V2 - SpanNetty - Re-enable ubuntu Test pipelines and fix failing TlsHandler tests
- [Bug 26682687](https://msazure.visualstudio.com/One/_workitems/edit/26682687): Gateway V2 - SpanNetty - Re-enable and fix intermittent TlsRead tests
- [Bug 26682694](https://msazure.visualstudio.com/One/_workitems/edit/26682694): Gateway V2 - SpanNetty - Re-enable .NET Fx pipeline and fix misc. intermittent failing tests and Netty.Transport.Tests timeout
- [Bug 26682630](https://msazure.visualstudio.com/One/_workitems/edit/26682630): Gateway V2 - SpanNetty - Re-enable and fix intermittent MqttServerAndClient End-To-End test
- [Bug 26682663](https://msazure.visualstudio.com/One/_workitems/edit/26682663): Gateway V2 - SpanNetty - Re-enable and fix intermittent EchoServerAndClient End-To-End test
We'll decide if these can in fact be done later, or have to be done before any merges. I've added priority to these based on my individual understanding, but we'll update as necessary.

Reply:
Mind adding them here? I know I'm a pain but when somebody sees this they don't know these exist nor context